freud/environment.pyx

```diff
@@ -664,7 +664,7 @@ cdef class EnvironmentCluster(_MatchEnv):
             warnings.warn(
                 "The global search option is deprecated and will be removed in "
                 "version 3.0. If you want this behavior, use a NeighborList "
-                "composed of all pairs of particles in the system as the `neighbors`.",
+                "composed of all pairs of particles in the system as the `cluster_neighbors`.",
                 FutureWarning
             )
 
```

Comment:
I remembered to test the branch after I pushed the changes. All the tests passed. I saw that there is a conflict waiting to be solved. I am not sure if that is caused by this?

Reply:
Don't worry about merge conflicts right now, I can fix those. It says there are conflicts in two flies, so its not just the line above.